File: app/src/main/java/org/mifos/mobile/ui/activities/SavingsAccountApplicationActivity.java
package org.mifos.mobile.ui.activities;

/*
 * Created by saksham on 30/June/2018
 */

import android.os.Bundle;

import org.mifos.mobile.R;
import org.mifos.mobile.ui.activities.base.BaseActivity;
import org.mifos.mobile.ui.enums.SavingsAccountState;
import org.mifos.mobile.ui.fragments.SavingsAccountApplicationFragment;

public class SavingsAccountApplicationActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_savings_account_application);
        setToolbarTitle(getString(R.string.apply_savings_account));
        showBackButton();
        replaceFragment(SavingsAccountApplicationFragment.newInstance(SavingsAccountState.CREATE,
                null), false, R.id.container);
    }
}

File: app/src/main/java/org/mifos/mobile/ui/activities/SavingsAccountApplicationActivity.kt
package org.mifos.mobile.ui.activities

import android.os.Bundle
import org.mifos.mobile.R
import org.mifos.mobile.databinding.ActivitySavingsAccountApplicationBinding
import org.mifos.mobile.ui.activities.base.BaseActivity
import org.mifos.mobile.core.model.enums.SavingsAccountState
import org.mifos.mobile.ui.savings_account.SavingsAccountApplicationFragment.Companion.newInstance

/*
* Created by saksham on 30/June/2018
*/
class SavingsAccountApplicationActivity : BaseActivity() {

    private lateinit var binding: ActivitySavingsAccountApplicationBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivitySavingsAccountApplicationBinding.inflate(layoutInflater)
        setContentView(binding.root)
        setToolbarTitle(getString(R.string.apply_savings_account))
        showBackButton()
        replaceFragment(
            newInstance(
                SavingsAccountState.CREATE,
                null,
            ),
            false,
            R.id.container,
        )
    }
}
